internal/bminventory/inventory.go

```diff
@@ -1464,6 +1464,9 @@ func (b *bareMetalInventory) createAndUploadNewImage(ctx context.Context, log lo
 			return err
 		}
 
+		details := b.getIgnitionConfigForLogging(ctx, infraEnv, log, imageType)
+		eventgen.SendIgnitionConfigImageGeneratedEvent(ctx, b.eventsHandler, *infraEnv.ID, details)
```

Comment:
I'm not sure if we really want this. But I had to either change the code or tests to get it working.

@filanov @avishayt what do you think? Should we still have an event here if we're not actually "generating" anything?

Reply:
Maybe changing the event, something like new image is ready or something like that.

Reply:
So create a new event? Right now the message is in the generated events code, right?

Reply:
yes